protocol/message.go

```diff
@@ -520,6 +524,36 @@ func (r *RegisterPreACState) Register() byte {
 	return PreACStateRegister
 }
 
+type RegisterACOperStatus struct {
```

Comment:
NB: I restored the code for register even if this remains unused, in case we want to use this later for anything or something...